app/views/kuroko2/job_definition_revisions/index.html.slim

```diff
@@ -0,0 +1,24 @@
+- content_for :title, "##{@definition.id} #{@definition.name} « Job Definitions"
```

Comment:
Definitions -> `Definition Revisions`?

Reply:
I fixed in https://github.com/cookpad/kuroko2/pull/110/commits/bc22e19d8e3c551858a905433fa0ba33ab69a5b2.